ports/esp32s2/common-hal/alarm/pin/PinAlarm.c

```diff
@@ -45,9 +60,248 @@ bool common_hal_alarm_pin_pin_alarm_get_value(alarm_pin_pin_alarm_obj_t *self) {
 }
 
 bool common_hal_alarm_pin_pin_alarm_get_edge(alarm_pin_pin_alarm_obj_t *self) {
-    return self->edge;
+    return false;
 }
 
 bool common_hal_alarm_pin_pin_alarm_get_pull(alarm_pin_pin_alarm_obj_t *self) {
     return self->pull;
 }
+
+gpio_isr_handle_t gpio_interrupt_handle;
+// Low and high are relative to pin number. 32+ is high. <32 is low.
+static volatile uint32_t low_pin_status = 0;
+static volatile uint32_t high_pin_status = 0;
```

Comment:
As noted below, maybe `pin_31_0_status` and `pin_63_32_status`, because `low` and `high` have two different meanings here: one is the pin number, and the other is the voltage at the pin.